extensions/resource-deployment/package.nls.json

```diff
@@ -1,23 +1,23 @@
 {
-	"extension-displayName": "Resource Deployment extension for Azure Data Studio",
-	"extension-description": "Provides a notebook-based experience to deploy SQL Server and other Azure Data Services",
+	"extension-displayName": "SQL Server Deployment extension for Azure Data Studio",
+	"extension-description": "Provides a notebook-based experience to deploy Microsoft SQL Server",
 	"deploy-sql-image-command-name": "Deploy SQL Server on Docker…",
 	"deploy-sql-bdc-command-name": "Deploy SQL Server Big Data Cluster…",
 	"deploy-resource-command-name": "Deploy SQL Server…",
-	"deploy-resource-command-category": "Resource Deployment",
+	"deploy-resource-command-category": "Deployment",
 	"resource-type-sql-image-display-name": "Container Image",
-	"resource-type-sql-image-description": "SQL Server container image on Docker",
+	"resource-type-sql-image-description": "Run SQL Server container image with Docker",
 	"resource-type-sql-bdc-display-name": "Big Data Cluster",
-	"resource-type-sql-bdc-description": "SQL Server Big Data Cluster",
+	"resource-type-sql-bdc-description": "SQL Server big data cluster allows you to deploy scalable clusters of SQL Server, Spark, and HDFS containers running on Kubernetes",
 	"version-display-name": "Version",
 	"sql-2017-display-name": "SQL Server 2017",
 	"sql-2019-display-name": "SQL Server 2019",
-	"sql-2017-docker-notebook": "./notebooks/sql2017-image-docker.ipynb",
-	"sql-2019-docker-notebook": "./notebooks/sql2019-image-docker.ipynb",
-	"sql-2019ctp30-display-name": "SQL Server 2019 Big Data Cluster CTP 3.0",
+	"sql-2017-docker-notebook": "./notebooks/docker/2017/deploy-sql2017-image.ipynb",
+	"sql-2019-docker-notebook": "./notebooks/docker/2019/deploy-sql2019-image.ipynb",
+	"sql-2019ctp31-display-name": "SQL Server 2019 Big Data Cluster CTP 3.1",
 	"bdc-deployment-target": "Deployment target",
 	"bdc-deployment-target-aks": "New Azure Kubernetes Service Cluster",
 	"bdc-deployment-target-existing-cluster": "Existing Kubernetes Cluster",
-	"bdc-ctp30-aks-notebook": "./notebooks/sql2019-ctp30-bdc-aks.ipynb",
-	"bdc-ctp30-existing-cluster-notebook": "./notebooks/sql2019-ctp30-bdc-existing-cluster.ipynb"
+	"bdc-ctp31-aks-notebook": "./notebooks/bdc/2019ctp30/deploy-bdc-aks.ipynb",
```

Comment:
The folder name should be updated to reflect the version bump, right?

2019ctp31